common/persistence/sql/sqlplugin/sqlite/plugin.go

```diff
@@ -204,5 +204,7 @@ func buildDSNAttr(cfg *config.SQL) (url.Values, error) {
 		// assume pragma
 		parameters.Add("_pragma", fmt.Sprintf("%s=%s", key, value))
 	}
+	// set time format
+	parameters.Add("_time_format", "sqlite")
```

Comment:
This line to make the SQLite driver module to format datetime using a supported SQLite format for datetime. Previously, without this setting, it was using `time.String()` to write datetime to the table.